src/views/virtualmachines/migrate/tabs/components/DiskTable.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { Dispatch, FC, SetStateAction } from 'react';
+
+import { IoK8sApiCoreV1PersistentVolumeClaim } from '@kubevirt-ui/kubevirt-api/kubernetes';
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { getName } from '@kubevirt-utils/resources/shared';
+import { getVolumes } from '@kubevirt-utils/resources/vm';
+import { readableSizeUnit } from '@kubevirt-utils/utils/units';
+import { Table, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table';
+
+import { getTableDiskData } from './utils';
+
+type DiskTableProps = {
+  pvcs: IoK8sApiCoreV1PersistentVolumeClaim[];
+  selectedPVCs: IoK8sApiCoreV1PersistentVolumeClaim[];
+  setSelectedPVCs: Dispatch<SetStateAction<IoK8sApiCoreV1PersistentVolumeClaim[]>>;
+  vm: V1VirtualMachine;
+};
+
+const columnNames = {
+  drive: 'Drive',
+  name: 'Name',
+  size: 'Size',
+  storageClass: 'Storage class',
+};
+
+const DiskTable: FC<DiskTableProps> = ({ pvcs, selectedPVCs, setSelectedPVCs, vm }) => {
```

Comment:
I think we already have a component called diskTable or something very similar in our VM details view/wizard view, can you rename this component?

Reply:
renamed `SelectMigrationDisksTable`